### PR TITLE
EZP-25114: Allows setting CSS custom class name for the ezimage field

### PR DESCRIPTION
**Pull request by clash82**
Link to Jira task: https://jira.ez.no/browse/EZP-25114

### CODE REVIEW

File: eZ/Bundle/EzPublishCoreBundle/Resources/views/content_fields.html.twig [hunk 1]
```diff
@@ -396,6 +396,7 @@
  #   - alias (image variation name). Defaults to "original" (e.g. image originally uploaded)
  #   - parameters.width. Allows forcing width of the image in the HTML
  #   - parameters.height. Allows forcing height of the image in the HTML
+ #   - parameters.class. Allows to set CSS custom class name of the image
```
Comment (DominikaEZ) [inline eZ/Bundle/EzPublishCoreBundle/Resources/views/content_fields.html.twig]: #   - parameters.class. Allows setting CSS custom class name for the image

Comment (clash82) [inline eZ/Bundle/EzPublishCoreBundle/Resources/views/content_fields.html.twig]: Fixed, thanks :)

**Comment by andrerom** [comment]
+1

**Comment by bdunogier** [comment]
Maybe we should consider a `spaceless` tag for this line, it's getting a bit long. That aside, +1.

**Comment by crevillo** [comment]
what about allow "classes" instead of "class"? i mean, you can of course pass "this is a big image" as class parameter, but at first "class" seems to expect only one... 

i'd got for parameter.classes and that being an array you should explode here...

**Comment by bdunogier** [comment]
Good point by @crevillo, I retract my +1 :-)

**Comment by crevillo** [comment]
to be honest i'm kinda "drupalthinking" here... :)

**Comment by clash82** [comment]
@crevillo I though `class` is a good name since we are setting `class` attribute but it makes no difference to me.

**Comment by crevillo** [comment]
yes, really the classes is because i think this should allow an array of strings instead an only string...

**Comment by bdunogier** [comment]
But why would you use an array rather than a string ?

`class="foo bar"`. Period, just like HTML. What would an array support that this wouldn't ?

**Comment by crevillo** [comment]
@bdunogier: Really Nothing. as said thinking in drupal way here. Suppose following case
 - Default ez publish install showing images like it does now
 - A bundle (aka plugin) that round all those images adding a class to all the image tags
 - Another one that allows you to open a cool window showing the image in their very big size. The related javascript added by this plugin needs a class placed in the image tags to work. 

so, something in the eZ kernel part will ask to the plugins if they have something to add to img class attribute. It will save all those in array and that array could be passed to the template or exploded before going to the template.... 

Same thing could be applied for body class, div classes... whatever...

**Comment by crevillo** [comment]
Anyway i'm just a thinking in a possible feature we still don't have. so i'm ok with this and if we do something else in the future we will have time to come back here :)
+1.

**Comment by clash82** [comment]
Guys, is it normal that some `ImageVariation` properties returns `null`?

screen shot 2015-11-20 at 11 59 53

**Comment by andrerom** [comment]
@clash82 I don't think @bdunogier noticed your question.

**Comment by bdunogier** [comment]
I indeed did not...

**Pull request merged**